consensus/types/src/attestation_safety.rs

```diff
@@ -0,0 +1,17 @@
+use crate::{Attestation, EthSpec, Hash256, PublicKeyBytes};
+
+#[derive(PartialEq, Debug)]
```

Comment:
looks like this file might have been inadvertently committed?

Reply:
thanks, removed in [fac7df7](https://github.com/dapplion/lighthouse/pull/36/commits/fac7df78fb3e6be799f79bdbe15464fe7833c727)